src/models/plateau/parser.rs

```diff
@@ -0,0 +1,87 @@
+use super::{Cell, Plateau, EMPTY, PLAYER1, PLAYER2};
+use std::convert::TryFrom;
+
+impl TryFrom<String> for Plateau {
+	type Error = String;
+
+	fn try_from(map: String) -> Result<Self, String> {
+		if map.is_empty() {
+			return Err(String::from("Map is empty"));
+		}
+		let width = match map.find(|c| c == '\n') {
+			Some(w) => w,
+			none => map.len(),
```

Comment:
Pretty sure this should be "None" - "none" would match anything in the same way that "_" does

Reply:
So it should have almost the same effect, except that the value "None" should get captured by "none"

Reply:
Fixed